server/routes/replayEventRouter.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const replayEventRouter = express.Router();
+
+const { getDomainEvents, dispatch } = require('../models/domain-event');
+
+const Session = require('../infra/database/Session');
+const { publishMessage } = require('../infra/messaging/RabbitMQMessaging');
+
+const { EventRepository } = require('../infra/database/PgRepositories');
+const { dictEventHandlers, handleEvent} = require('../services/event-handlers');
+
+const processEvent = (event, executor) => {

```

Comment:
we could eliminate this `processEvent` function since the executor has a try catch implemented.